app/PiCalculator/pi.js

```diff
@@ -0,0 +1,222 @@
+var numOfPointsToTryPerClient = 100000000000;
+var batchSizePerUpdate = 5000000;
+
+PIData = new Mongo.Collection('pidata');
+var n = 5;
+
+var signedInWithFacebook = false;
+var signedInWithGoogle = false;
+var signedInWithLocalAccount = false;
+
+var initialized = false;
+
+var getCurrentTime = function () {
+    var currentDate = new Date();
+    var currentTime = currentDate.getHours() * 3600 + currentDate.getMinutes() * 60 + currentDate.getSeconds();
+    return currentTime;
+};
+
+function float2int(value) {
+    return value | 0;
+}
+
+var getId = function () {
+    return (((1+Math.random())*0x10000)|0).toString(16).substring(1);
```

Comment:
Something like `Math.random().toString(16).substr(2,4)` might be more readable here